### PR TITLE
Added styles to user component.

### PR DESCRIPTION
**Pull request by anchounio**

### CODE REVIEW

File: src/components/core/user.js [hunk 1]
```diff
@@ -1,8 +1,16 @@
+import './user.scss';
+
 export function User() {
     return (
         <div className="app-user">
-            <img src="./public/assets/pepe.png" alt="el pepe" />
-            <span>Pepe</span>
+            <span className="app-user__container-image">
```
Comment (amorga97) [inline src/components/core/user.js:6]: If it is not necessary, remove it

File: src/components/core/user.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+                <img
+                    className="app-user__container-image__image"
```
Comment (amorga97) [inline src/components/core/user.js:8]: Change it to app-user__image

**Review by amorga97 — CHANGES_REQUESTED** [review]

File: src/components/core/user.js (hunk 1 above)
Comment (anchounio) [reply src/components/core/user.js]: done

File: src/components/core/user.js (hunk 2 above)
Comment (amorga97) [reply src/components/core/user.js]: Good

**Review by amorga97 — APPROVED** [review]

**Review by sebsVR — APPROVED** [review]
Great!

**Pull request merged**